serde_derive_implementation/src

```diff
@@ -0,0 +1 @@
+../serde_derive/src/implementation/
```

Comment:
Is this a symbolic link?

Reply:
yup it was, just like for `serde_derive_internals `
See: https://github.com/serde-rs/serde/blob/master/serde_derive_internals/src